charts/thin/templates/fuse/daemonset.yaml

```diff
@@ -102,8 +104,7 @@ spec:
           - name: thin-fuse-mount
             mountPath: {{ .Values.fuse.targetPath | dir }}
             mountPropagation: Bidirectional
-          - mountPath: /etc/fluid/config.json
-            subPath: config.json
+          - mountPath: /etc/fluid/config
```

Comment:
Is this a breaking change?

Reply:
Yes, the previous method of using subpath to mount configmap cannot be updated in the fuse pod.